src/mlpack/tests/init_rules_test.cpp

```diff
@@ -126,6 +129,35 @@ BOOST_AUTO_TEST_CASE(OivsInitTest)
   BOOST_REQUIRE_EQUAL(1, 1);
 }
 
+BOOST_AUTO_TEST_CASE(XavierInitTest)
+{
+  arma::mat weights;
+  //RandomSeed(21);
+  XavierInit xavierUniform;
+  XavierInit xavierNormal;
+  xavierUniform.Initialize<XavierUniform>(weights, 100, 100);
```

Comment:
We require Visual Studio 2015 minimum, since earlier versions don't support a lot of C++11 features that we need.  The comment that you referenced is about VS2012, so I wouldn't worry about it.